app/controllers/wikis_controller.rb

```diff
@@ -25,7 +26,11 @@ def create
     @wiki = Wiki.new
     @wiki.title = params[:wiki][:title]
     @wiki.body = params[:wiki][:body]
-    @wiki.private = params[:wiki][:private]
+
+    if params[:wiki][:private]
+      @wiki.private = params[:wiki][:private]
+    end
```

Comment:
Just FYI, a slightly better, cleaner way of checking for the presence of the parameter would be:

```ruby
unless params[:wiki][:private].nil?
  @wiki.private = params[:wiki][:private]
end
```

Checking for "nilness" allows for `false` to be still considered an actual value for the parameter. Right now it would be discarded, which is *fine* because the default value of the private attribute is `false` anyway, but…